packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -414,6 +438,46 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
         }
     }
 
+    private onAndroidInputEvent(event: PluginInputEvent) {
+        const wordBeforCursor = this.getInlineElementBeforeCursor(event);
+        this.newInputLength = wordBeforCursor ? wordBeforCursor.length : 0;
+
+        if (
+            this.newInputLength < this.currentInputLength ||

```

Comment:
if I type on a long node, then move cursor to a short one then continue type, it is possible this still get triggered. In that case I think it is wrong.

Reply:
Does this mean the short node will be removed? Can we detect when the cursor has been moved (also, is this by mouse or keyboard)?

Reply:
Should be fixed if the latest change works as expected